docs/source/api.rst

```diff
@@ -36,6 +34,15 @@ API
    Client.upload_file
    Client.who_has
 
+.. currentmodule:: distributed.recreate_exceptions
```

Comment:
@mrocklin This makes the two methods appear below the others, which isn't optimal, but probably OK.